app/routes/user/password.server.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+import { ChangePasswordCommand } from '@aws-sdk/client-cognito-identity-provider'
+
+import { getOpenIDClient, storage } from '../__auth/auth.server'
+import { getUser, parseTokenSet } from '../__auth/user.server'
+import { maybeThrow } from './cognito.server'
+import { cognito } from '~/lib/cognito.server'
+
+export module PasswordManager {
+  async function updatePasswordFunc(
```

Comment:
Get rid of the unnecessary module and inner closure. Let's just have one function `updatePassword`, at _file_ scope.